src/mem/cache/replacement_policies/replaceable_entry.hh

```diff
@@ -65,12 +65,12 @@ class ReplaceableEntry
     /**
      * Set to which this entry belongs.
      */
-    uint32_t _set;
+    uint32_t _set = 0;
```

Comment:
Nit: use list initialization instead of assignment

Reply:
Fixed in the new push.